vllm/attention/selector.py

```diff
@@ -88,13 +89,12 @@ def get_global_forced_attn_backend() -> Optional[_Backend]:
 
 @lru_cache(maxsize=None)
 def get_attn_backend(
-    num_heads: int,
     head_size: int,
-    num_kv_heads: int,
     sliding_window: Optional[int],
     dtype: torch.dtype,
     kv_cache_dtype: Optional[str],
     block_size: int,
+    is_attention_free: bool,
```

Comment:
Could you set this to False by default? Or do you think every usage should be explicit about this?

Reply:
I think most (maybe all but one?) callers of the selector should have to be explicit about this